LiveSplit.AutoSplitters.xml

```diff
@@ -15248,6 +15248,16 @@
     <AutoSplitter>
         <Games>
             <Game>Red Ball</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/rb1improved.asl</URL>
+        </URLs>
+        <Type>Script</Type>
+        <Description>Perfectly splits upon completing each level. Click "Website" to download the right game version. (By Maximum)</Description>
+        <Website>https://drive.google.com/file/d/12X0J05rZ6gRnPgybDJuz56xGGnxjaOev/view?usp=sharing</Website>
```

Comment:
Is this a link to the game?

Reply:
Is this being hosted on your Google Drive? Is this copyrighted content, and if so, did you obtain permission to host it?

Reply:
Yes, this is a link to a modified version of the game, and it is hosted on my google drive. We have been given permission from the game's creator to use the source code for any non-commercial purposes. ![Proof](https://github.com/LiveSplit/LiveSplit.AutoSplitters/assets/58001630/618db0c8-503e-4b3b-923f-aaeaec83f4b0)

Reply:
Great, thank you for clarifying